src/status_im/ui/components/icons/icons.clj

```diff
@@ -6,7 +6,7 @@
 
 (defn require-icon
   [el]
-  (let [s (str "../resources/images/icons/" el ".png")
+  (let [s (str "." "../resources/images/icons/" el ".png")
```

Comment:
This `str` call concatenates to a path starting with 3 dots, is this really correct?

Reply:
nope, there should be no changes to this file. will revert 👍